macros/src/lib.rs

```diff
@@ -277,3 +279,117 @@ pub fn modal(input: TokenStream) -> TokenStream {
         Err(e) => e.write_errors().into(),
     }
 }
+
+/**
+Use this macro on an impl block to implement the CommandGroup trait.
```

Comment:
What `CommandGroup` trait?

Reply:
Oh yeah 😅 that doesn't exist. Should I add it?

Reply:
Yes, more details in another review comment.